src/Microsoft.AspNet.Mvc.Core/Formatters/InputFormatter.cs

```diff
@@ -67,42 +67,48 @@ public virtual bool CanRead(InputFormatterContext context)
                 return false;
             }
 
-            return SupportedMediaTypes
-                            .Any(supportedMediaType => supportedMediaType.IsSubsetOf(requestContentType));
+            return SupportedMediaTypes.Any(supportedMediaType => supportedMediaType.IsSubsetOf(requestContentType));
         }
 
         /// <summary>
-        /// Returns a value indicating whether or not the given type can be read by this serializer.
+        /// Determines whether this <see cref="InputFormatter"/> can deserialize an <c>object</c> of the given
+        /// <paramref name="type"/>.
         /// </summary>
-        /// <param name="type">The type of object that will be read.</param>
-        /// <returns><c>true</c> if the type can be read, otherwise <c>false</c>.</returns>
+        /// <param name="type">The <see cref="Type"/> of <c>object</c> that will be read.</param>
+        /// <returns><c>true</c> if the <paramref name="type"/> can be read; <c>false</c> otherwise.</returns>
```

Comment:
Subtracting the markup:

> true if the type can be read, otherwise false.
> true if the type can be read; false otherwise.

A lot of these doc changes are improvements, but very tiny ones